diff -up libebml-1.4.4/src/EbmlUnicodeString.cpp.utf8cpp libebml-1.4.4/src/EbmlUnicodeString.cpp
--- libebml-1.4.4/src/EbmlUnicodeString.cpp.utf8cpp	2022-10-11 13:11:14.129418820 +0200
+++ libebml-1.4.4/src/EbmlUnicodeString.cpp	2022-10-11 13:11:44.405608490 +0200
@@ -40,7 +40,7 @@
 
 #include "ebml/EbmlUnicodeString.h"
 
-#include "lib/utf8-cpp/source/utf8/checked.h"
+#include <utf8/checked.h>
 
 namespace libebml {
 
